contracts/AllowedStrategy.sol

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import "./common/UsingRegistryUpgradeable.sol";
+import "./common/UUPSOwnableUpgradeable.sol";
+import "./interfaces/IAccount.sol";
+import "./interfaces/IGroupHealth.sol";
+import "./interfaces/IManager.sol";
+import "./Managed.sol";
+
+contract AllowedStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    /**
+     * @notice Allowed strategies (validator groups) that can be chosen to voted on.
```

Comment:
Why are you saying strategies = validator groups? Isn't the default strategy an allowed strategy but not a validator group per se?